lib/handbook/addon/docs/resources/template.md

```diff
@@ -12,3 +12,17 @@
 ## Ember + TypeScript
 - [ember-cli-typescript](https://github.com/typed-ember/ember-cli-typescript/blob/master/README.md) - what makes it all possible
 - [Typing Your Ember](https://www.chriskrycho.com/typing-your-ember.html) - a blog series by [@chriskrycho](https://www.chriskrycho.com/about.html)
+
+## Various Addons
+- [ember-a11y-testing](https://github.com/ember-a11y/ember-a11y-testing) - Catching easy-to-miss, easy-to-detect accessibility problems
+- [ember-cli-addon-docs](https://github.com/ember-learn/ember-cli-addon-docs) - That's this handbook
+- [ember-cli-mirage](http://www.ember-cli-mirage.com) - Fake API responses for easy testing and development
+- [ember-concurrency](http://ember-concurrency.com/docs/introduction/) - Tasks for asynchronous actions
+- [ember-css-modules](https://github.com/salsify/ember-css-modules) - Don't style globally when you can style locally
+- [ember-decorators](https://github.com/ember-decorators/ember-decorators) - Save a lot of trouble coding… with decorators!
+- [ember-engines](https://github.com/ember-engines/ember-engines) - How we separate code now
+- [ember-feature-flags](https://github.com/kategengler/ember-feature-flags) - For waffle feature flags
```

Comment:
I mostly added that because we often talk about "waffle flags" instead of "feature flags`" and just saying "For feature flags" would be super redundant with the name.

Reply:
The Ember app doesn't refer to them as waffle flags anywhere and even the API just calls them `active_flags` (at least in the endpoint we use). IMHO we should stop calling them waffle flags because waffle is a back-end implementation detail.

Reply:
Also, I'd like to start using `ember-feature-flags` for more than just flags we get from the API. For example, I'd like to start using it for https://www.branchbyabstraction.com